src/pages/ClusterPage/explore.js

```diff
@@ -121,15 +124,20 @@ export default class ExploreCluster extends Component {
 		const arcURL = this.state.arc.url
 			? this.state.arc.url.slice(0, -1)
 			: '';
-		let mainURL = 'https://arc-dashboard.appbase.io';
+		let mainURL = this.state.redirectToBilling
```

Comment:
this can be `this.state.cluster ? ... : ....`

Reply:
Actually this is for another use case. It's like in dashboard, when you click the red "upgrade now" button in header, you need to redirected into arc's cluster billing page.

Reply:
It needs to be redirected to the '/billing' page instead of '/' root page of arc-dashboard

Reply:
@lakhansamani  Got your point